sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsTraceExporter.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals;
+using OpenTelemetry;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics
+{
+    /// <summary>
+    /// Empty Exporter. This is used to enable the Manager Singleton.
+    /// </summary>
+    public class LiveMetricsTraceExporter : BaseExporter<Activity>
```

Comment:
Why this is needed?

Reply:
This is the entry point for my POC. Adding this to a TracerProvider initializes the Manager Singleton.